<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lobby</class>
 <widget class="QWidget" name="Lobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Attorney Online 2</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QStackedWidget" name="stacked_widget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">#server_list_page {
  background-image: url(:/base/themes/default/lobbybackground.png);
}</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="server_list_page">
        <widget class="QListWidget" name="server_list">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>125</y>
           <width>286</width>
           <height>240</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0, 0, 0, 0); font: bold; border: none;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="public_servers">
         <property name="geometry">
          <rect>
           <x>46</x>
           <y>88</y>
           <width>114</width>
           <height>30</height>
          </rect>
         </property>
         <property name="accessibleName">
          <string>Public Servers</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  border-image: url(:/base/themes/default/publicservers.png);
}
:checked {
  border-image: url(:/base/themes/default/publicservers_selected.png);
}</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="refresh">
         <property name="geometry">
          <rect>
           <x>56</x>
           <y>381</y>
           <width>132</width>
           <height>28</height>
          </rect>
         </property>
         <property name="accessibleName">
          <string>Refresh</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  border-image: url(':/base/themes/default/refresh.png');
}
:pressed {
  border-image: url(':/base/themes/default/refresh_pressed.png');
}</string>
         </property>
        </widget>
        <widget class="QTextBrowser" name="description">
         <property name="geometry">
          <rect>
           <x>337</x>
           <y>109</y>
           <width>173</width>
           <height>245</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0, 0, 0, 0); color: white;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="favorites">
         <property name="geometry">
          <rect>
           <x>164</x>
           <y>88</y>
           <width>114</width>
           <height>30</height>
          </rect>
         </property>
         <property name="accessibleName">
          <string>Favorites</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  border-image: url(:/base/themes/default/favorites.png);
}
:checked {
  border-image: url(:/base/themes/default/favorites_selected.png);
}</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="player_count">
         <property name="geometry">
          <rect>
           <x>336</x>
           <y>91</y>
           <width>173</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: bold; color: white;</string>
         </property>
         <property name="text">
          <string>Offline</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="about">
         <property name="geometry">
          <rect>
           <x>428</x>
           <y>1</y>
           <width>88</width>
           <height>21</height>
          </rect>
         </property>
         <property name="accessibleName">
          <string>About...</string>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(':/base/themes/default/about.png');</string>
         </property>
        </widget>
        <widget class="QPushButton" name="connect">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>332</x>
           <y>381</y>
           <width>132</width>
           <height>28</height>
          </rect>
         </property>
         <property name="accessibleName">
          <string>Connect</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  border-image: url(':/base/themes/default/connect.png');
}
:pressed {
  border-image: url(':/base/themes/default/connect_pressed.png');
}</string>
         </property>
        </widget>
        <widget class="QPushButton" name="add_to_fav">
         <property name="geometry">
          <rect>
           <x>194</x>
           <y>381</y>
           <width>132</width>
           <height>28</height>
          </rect>
         </property>
         <property name="accessibleName">
          <string>Add to Favorites</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  border-image: url(':/base/themes/default/addtofav.png');
}
:pressed {
  border-image: url(':/base/themes/default/addtofav_pressed.png');
}</string>
         </property>
        </widget>
        <widget class="QLabel" name="version">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>1</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>AO version</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="AOServerChat" name="chat" native="true">
         <property name="geometry">
          <rect>
           <x>2</x>
           <y>443</y>
           <width>515</width>
           <height>222</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QTextBrowser, QLineEdit { background-color: rgb(0, 0, 0, 0); border: none; }
QLineEdit { color: white; }</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="loading_page">
        <property name="styleSheet">
         <string notr="true">#loading_page {
  background-image: url(:/base/themes/default/loadingbackground.png);
  background-position: bottom;
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="loading_background">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="4" alignment="Qt::AlignBottom">
          <widget class="QPushButton" name="cancel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="loading_text">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>2</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="text">
            <string>Loading...</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="1" colspan="2">
          <widget class="QProgressBar" name="progress_bar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="menu_file_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AOServerChat</class>
   <extends>QWidget</extends>
   <header>widgets/aoserverchat.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
